.gitignore

```diff
@@ -87,3 +87,6 @@ packages/system-tests/project_*
 
 # Node 8 files (not yet used)
 package-lock.json
+
+# External extension dependencies
```

Comment:
We will install this at the top-level so we should definitely ignore it from being committed. We need it to be at the same level as the other extensions.

The downside of this is that now we are also loading all the other extensions from the /packages directory so there will be some additional overhead.